modules/pel/peltool/user_header.py

```diff
@@ -36,9 +36,18 @@ def __init__(self, stream: DataStream, sectionID: int, sectionLen: int,
         self.actionFlags = 0
         self.states = 0
 
+    def isHidden(self) -> bool:
+        return self.actionFlags & actionFlagsBits["hiddenFlagBit"]
+
     def isServiceable(self) -> bool:
-        # consider it a serviceable PEL if not an info or recovered error
-        return self.eventSeverity != 0x00 and self.eventSeverity != 0x10
+        if self.eventSeverity != 0x00:  ## if it isn't Informational
```

Comment:
Pls [SeverityValues](https://github.com/ibm-openbmc/openpower-pel-parsers/blob/peltool-testing/modules/pel/peltool/pel_types.py#L5) enum instead of using/harding number which is not readable.

Code should not exist max character  (75) per line